### PR TITLE
Parser First Commit

### PR DESCRIPTION
**Pull request by tavisca-ishan**
Please Review First Review for Parser

### CODE REVIEW

File: 01 Operator Overloading/OperatorOverloading.dbl/Fetch.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.IO;
+namespace OperatorOverloading.dbl
+{
+    // following class fetches a Web Page
+
+    class WebFetch
```
Comment (apalsapure) [inline 01 Operator Overloading/OperatorOverloading.dbl/Fetch.cs:12]: Read about WebClient

**Comment by apalsapure** [comment]
Can't review your pull request as it can not be merged.

**Comment by apalsapure** [comment]
Lots of naming convention issues. The money object should have `Convert` method which will return `Money` object.

**Comment by tavisca-ishan** [comment]
hi,
I am not able to resolve these conflicts.Wish to seek your help!

**Comment by tavisca-ishan** [comment]
@apalsapure  Please Review Parser Update

**Comment by tavisca-ishan** [comment]
@apalsapure please review the commit for parser with app.config updated

**Comment by tavisca-ishan** [comment]
JSONParser Final Commit

**Pull request closed without merging**